controllers/controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../models/model")
+
+exports.getTopics = (req,res, next)=>{
+    selectTopics().then((topics)=>{
+        res.status(200).send(topics)
```

Comment:
Ideally you should be sending your response in an object on an appropriate key 🔑